### PR TITLE
Lo-fi Footer Section

### PR DESCRIPTION
**Pull request by rmehri01**
## :construction_worker: Changes

- Changed footer text to match.
- Added desktop and mobile responsive css.

## :flashlight: Testing Instructions

- Try it out in amplify and use your browser's responsive mode to view it on a phone or tablet.

### CODE REVIEW

**Comment by kozr** [comment]
Can you have info@nwplus.io instead of hello@nwplus.io?

**Comment by kozr** [comment]
image
When you squeeze it a lot, "Become a sponsor" splits into two. If it's a quick fix then go for it but I think it's a very small chance that someone will encounter that.

File: components/Footer.vue [hunk 1]
```diff
@@ -116,29 +116,78 @@
...
-      default: 'Organized and held by nwPlus. Copyright © nwHacks 2019'
-    }
-  },
-  data() {
-    return {
-      splitText: this.text.split('.')
-    }
-  }
-}
+export default {}
```
Comment (kozr) [inline components/Footer.vue:129]: What is this exporting?

**Comment by rmehri01** [comment]
@kozr What should happen then, should it become a column when its that narrow?

**Comment by kozr** [comment]
Yeah it would be nice to have it on a separate row

File: components/Footer.vue (hunk 1 above)
Comment (rmehri01) [reply components/Footer.vue:129]: Oh my bad, the stuff from before wasn't being used so I can just remove it.

**Pull request merged**